backend/app/models/experiment_template.py

```diff
@@ -189,37 +189,37 @@ def is_same_environment(
             ]
         ) == len(self.environment_attribute_names)
 
-    def update_non_environment(self, new_template: ExperimentTemplate) -> None:
-        self.name = new_template.name
-        self.description = new_template.description
-
     @classmethod
     async def update_template(
         cls,
-        old_template: ExperimentTemplate,
+        original_template: ExperimentTemplate,
         experiment_template_req: ExperimentTemplateCreate,
         editable_environment: bool,
         editable_visibility: bool,
     ) -> ExperimentTemplate | None:
-        same_environment = old_template.is_same_environment(experiment_template_req)
-        same_visibility = old_template.is_public == experiment_template_req.is_public
+        same_environment = original_template.is_same_environment(
+            experiment_template_req
+        )
+        same_visibility = original_template.public == experiment_template_req.public
         new_template = ExperimentTemplate(
-            **experiment_template_req.dict(), created_by=old_template.created_by
+            **experiment_template_req.dict(), created_by=original_template.created_by
         )
         template_to_return = None
 
         if same_environment and (same_visibility or editable_visibility):
             # We modify only name & descr (+ maybe visibility)
-            old_template.update_non_environment(new_template)
-            old_template.is_public = experiment_template_req.is_public
-            old_template.updated_at = new_template.updated_at
-            template_to_return = old_template
+            original_template.name = new_template.name
+            original_template.description = new_template.description
+            original_template.public = experiment_template_req.public
```

Comment:
Can this be actually changed to `original_template.public = new_template.public`?